src/com/amplitude/api/AmplitudeClient.java

```diff
@@ -936,6 +922,14 @@ static boolean upgradePrefs(Context context, String sourcePkgName, String target
                 target.putBoolean(Constants.PREFKEY_OPT_OUT,
                         source.getBoolean(sourcePkgName + ".optOut", false));
             }
+            if (source.contains(sourcePkgName + ".lastEventId")) {
```

Comment:
this doesn't need to happen, since this is part of a migration path that didn't have these values